manifests/init.pp

```diff
@@ -37,6 +37,9 @@
 #
 # $package_names::      Packages that this module ensures are present instead of the default
 #
+# $enable_ostree::      Boolean to enable ostree plugin. This requires existence of a ostree install.Defaults to false
```

Comment:
I don't mind, but in that case I'd at least place a space between the dot and Defaults for readability.